ansible/roles/base/tasks/compile.yml

```diff
@@ -0,0 +1,17 @@
+- name: clone firecracker repo
+  git:
+    repo: 'https://github.com/firecracker-microvm/firecracker'
+    dest: src/firecracker
+    force: yes
+
+- name: check if binary exists
+  stat:
+    path: src/firecracker/build/release/firecracker
+  register: binary_exists
+
+- name: build firecracker from source
+  shell: src/firecracker/tools/devtool build --release
+  when: binary_exists.stat.exists == False
+
+- include: download.yml
```

Comment:
Don't put it here. The downloaded files are used in the execute.yml, yet they are children of a sibling tree (compile.yml).

Reply:
Done